index.js

```diff
@@ -189,21 +202,57 @@ Parser.prototype._flush = function(done) {
     var self = this;
     var base = path.resolve( self.base, self.output );
     var translationsHash = {};
+    var translationsHashCommented = {};
+
+
+
+    // merge file paths
+    // ================
+    var tempHash = {};
+    _.map(self.translations, (_, index) => {
+        var translation = self.translations[index];
+
+        if (tempHash.hasOwnProperty(translation.key)) {
+            tempHash[translation.key] = tempHash[translation.key].concat(translation.paths);
+        } else {
+            tempHash[translation.key] = translation.paths;
+        }
 
+        // keep file paths in predictable order
+        tempHash[translation.key].sort();
 
+        return _;
```

Comment:
I tend not to return anything if we don't need it. I think it would be more consistent with the rest of the project.